instance/models/openedx_appserver.py

```diff
@@ -87,9 +88,12 @@ class Meta:
 
     # Misc settings:
     use_ephemeral_databases = models.BooleanField()
-    github_admin_organization_name = models.CharField(
-        max_length=200, blank=True, default=settings.DEFAULT_ADMIN_ORGANIZATION,
-        help_text="GitHub organization whose users will be given SSH access to this instance's VMs",
+    github_admin_users = JSONField(
```

Comment:
> That said, I don't have a strong preference either way, and since you seem to prefer the other approach, I can just as well go with it. It actually makes the code a bit simpler, and reduces the number of queries to the Github API.

Let's do that then; I think it'll give us a cleaner, more explicit codebase that'll be easier to maintain.